cmd/clusterctl/validation/validate_pods.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+	"io"
+
+	"golang.org/x/net/context"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func ValidatePods(w io.Writer, c client.Client) error {
+	fmt.Fprintf(w, "Validating pods\n")
+
+	pods, err := getPods(c, "kube-system")
```

Comment:
Can the namespace be passed in instead of hard code here?

Reply:
I have passed `kube-system` from `cmd/validate_cluster.go` instead if this is fine https://github.com/kubernetes-sigs/cluster-api/pull/679/commits/c77856e21219fa6c6a4969b810168786e6e3978b#diff-6334f0ae8a2b2bebd8b635352f2a9426R79